moto/secretsmanager/models.py

```diff
@@ -29,51 +30,57 @@ class SecretsManagerBackend(BaseBackend):
     def __init__(self, region_name=None, **kwargs):
         super(SecretsManagerBackend, self).__init__()
         self.region = region_name
-        self.secret_id = kwargs.get('secret_id', '')
-        self.name = kwargs.get('name', '')
-        self.createdate = int(time.time())
-        self.secret_string = ''
-        self.rotation_enabled = False
-        self.rotation_lambda_arn = ''
-        self.auto_rotate_after_days = 0
-        self.version_id = ''
+        self.secrets = {}
 
     def reset(self):
         region_name = self.region
         self.__dict__ = {}
         self.__init__(region_name)
 
     def _is_valid_identifier(self, identifier):
-        return identifier in (self.name, self.secret_id)
+        if identifier in self.secrets:
+            return True
 
     def get_secret_value(self, secret_id, version_id, version_stage):
 
         if not self._is_valid_identifier(secret_id):
             raise ResourceNotFoundException()
 
+        secret = self.secrets[secret_id]
+
         response = json.dumps({
-            "ARN": secret_arn(self.region, self.secret_id),
-            "Name": self.name,
-            "VersionId": "A435958A-D821-4193-B719-B7769357AER4",
-            "SecretString": self.secret_string,
+            "ARN": secret_arn(self.region, secret['secret_id']),
+            "Name": secret['name'],
+            "VersionId": secret['version_id'],
+            "SecretString": secret['secret_string'],
             "VersionStages": [
                 "AWSCURRENT",
             ],
-            "CreatedDate": "2018-05-23 13:16:57.198000"
+            "CreatedDate": secret['createdate']
         })
 
         return response
 
     def create_secret(self, name, secret_string, **kwargs):
 
-        self.secret_string = secret_string
-        self.secret_id = name
-        self.name = name
+        self.generated_version_id = str(uuid.uuid4())
```

Comment:
why is this stored as an instance variable? Shouldn't it be just a regular variable?

Reply:
Good catch! Switching to a regular variable